scenarios/18_Start_of_the_Quest.cfg

```diff
@@ -257,7 +257,11 @@
             side=1
             [modifications]
                 {TRAIT_INTELLIGENT}
+#ifdef NIGHTMARE
```

Comment:
And here, no debuff please.

Reply:
[086c1d2](https://github.com/nemaara/A_New_Order/pull/53/commits/086c1d25c13ea2fc5549ae746b68eb2476cc283a).